Api/WMC/WMC1/Request.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+namespace baibaratsky\WebMoney\Api\WMC\WMC1;
+
+use baibaratsky\WebMoney\Api\WMC;
+use baibaratsky\WebMoney\Exception\ApiException;
+use baibaratsky\WebMoney\Request\RequestValidator;
+use baibaratsky\WebMoney\Signer;
+
+/**
+ * Class Request
+ *
+ * @link http://wiki.wmtransfer.com/projects/webmoney/wiki/Interface_WMC1
+ */
+class Request extends WMC\Request
+{
+
+    /** @var string payment/purse */
+    protected $payeePurse;
+
+    /** @var string payment/@currency */
+    protected $currency;
+
+    /** @var float payment/price */
+    protected $price;
+
+    /** @var int payment/phone */
+    protected $phone = '';
+
+    /**
+     * @param string $authType
+     *
+     * @throws ApiException
+     */
+    public function __construct($authType = self::AUTH_CLASSIC)
+    {
+        switch ($authType) {
+            case self::AUTH_CLASSIC:
+                $this->url = 'https://transfer.gdcert.com/ATM/Xml/PrePayment1.ashx';
+                break;
+
+            case self::AUTH_LIGHT:
+                $this->url = 'https://transfer.gdcert.com/ATM/Xml/PrePayment1.ashx';
+                $this->signature = base64_encode(
```

Comment:
This is not right way to initialise signature, I should move it to cert() method.